### PR TITLE
change format to fstring literals

### PR DESCRIPTION
**Pull request by yukilikespie**

### CODE REVIEW

File: tor_ocr/errors.py [hunk 1]
```diff
@@ -21,11 +21,7 @@ def __init__(self, result):
         super(OCRError, self).__init__(
             self.PAGE_EXIT_CODES.get(
                 result['exit_code'],
-                "Unrecognized error: Code {0}, {1}: {2}".format(
-                    result['exit_code'],
-                    result['error_message'],
-                    result['error_details']
-                )
+                f"Unrecognized error: Code {result['exit_code']}, {result['error_message']}: {result['error_details']}"
```
Comment (perryprog) [inline tor_ocr/errors.py]: This line is too long

File: tor_ocr/main.py [hunk 2]
```diff
@@ -138,12 +138,7 @@ def decode_image_from_url(url, overlay=False, api_key=__OCR_API_KEY__):
 
     if not result.ok:
         logging.error(
-            'ERROR {code} with OCR:\n\nHEADERS:\n{headers}\n\nBODY:\n{body}'
-            ''.format(
-                headers=repr(result.headers),
-                body=repr(result.text),
-                code=result.status_code,
-            )
+            f'ERROR {result.status_code} with OCR:\n\nHEADERS:\n{repr(result.headers)}\n\nBODY:\n{repr(result.text)}'
```
Comment (perryprog) [inline tor_ocr/main.py]: This line is too long

**Review by perryprog — CHANGES_REQUESTED** [review]
See my review in the main tor repo. Again, I'm not sure if we're using single or double quotes, but we'll figure that out depending on what Ghost/Joker think

File: tor_ocr/errors.py [hunk 3]
```diff
@@ -21,11 +21,8 @@ def __init__(self, result):
         super(OCRError, self).__init__(
             self.PAGE_EXIT_CODES.get(
                 result['exit_code'],
-                "Unrecognized error: Code {0}, {1}: {2}".format(
-                    result['exit_code'],
-                    result['error_message'],
-                    result['error_details']
-                )
+                f"Unrecognized error: Code {result['exit_code']}, "
+                f"{result['error_message']}: {result['error_details']}"
```
Comment (perryprog) [inline tor_ocr/errors.py:25]: Single quotes here and here

**Review by perryprog — CHANGES_REQUESTED** [review]

**Review by perryprog — APPROVED** [review]
I'm stupid nvm

**Review by TheLonelyGhost — APPROVED** [review]
Nothing blatantly wrong, though I'm unsure if following our pre-existing style in a part where we nest `result['error_message']` inside of an `f`-string would make it react differently if the `f`-string was with single- vs. double-quotes.

Not a big deal. It doesn't matter so much to me that I'd be willing to research it. 👍 Great work! Thanks!

**Pull request merged**